src/main/java/org/spongepowered/common/interfaces/entity/IMixinEntity.java

```diff
@@ -130,6 +130,8 @@ default void supplyVanillaManipulators(List<DataManipulator<?, ?>> manipulators)
 
     void setLocationAndAngles(Transform<World> transform);
 
+    boolean hasGravity();
```

Comment:
This isn't needed as it should be overriding `Entity`'s method, not making a new one.